.travis.yml

```diff
@@ -5,6 +5,7 @@ python:
     - "3.4"
 
 env:
+    - SECRET_KEY=djkk\&8izc3tmgva\(m\$%u_zvjk8%f@-6w\(d-z68@m-@sh3nz\#pi
     - DJANGO_VERSION=1.8
 
 install:
```

Comment:
I really don't think that's the issue. `travis` is to be installed locally for encryption and other stuff.